src/Sulu/Bundle/WebsiteBundle/EventSubscriber/AnalyticsSerializeEventSubscriber.php

```diff
@@ -50,5 +50,11 @@ public function onPostSerialize(ObjectEvent $event)
             $value->domains = true;
             $event->getVisitor()->visitProperty($metadata, $value, $event->getContext());
         }
+
+        $content = $analytics->getContent();
```

Comment:
the new serializer seems to detect always an array for content. but it could be sometimes "string" also. this lasts into failing tests. this workaround checks the type before serialization.